document_page/models/document_page_history.py

```diff
@@ -33,22 +30,38 @@ class DocumentPageHistory(models.Model):
     _order = 'id DESC'
     _rec_name = "create_date"
 
-    page_id = fields.Many2one('document.page', 'Page')
+    page_id = fields.Many2one('document.page', 'Page', ondelete='cascade')
     summary = fields.Char('Summary', index=True)
     content = fields.Text("Content")
-    create_date = fields.Datetime("Date")
-    create_uid = fields.Many2one('res.users', "Modified By")
+    diff = fields.Text(compute='_compute_diff')
+
+    @api.multi
```

Comment:
same here

Reply:
@ivantodorovich I think we still need an api.depends here, no?

Reply:
Since it's not stored, it's actually not needed.
Function gets called every time you access the field.

Reply:
Just in case, just added it.